### PR TITLE
Add Lighthouse back to Simorgh to enable local lighthouse testing

### LINKED ISSUE #1694: Add Lighthouse back to Simorgh to enable local lighthouse testing
**Is your feature request related to a problem? Please describe.**
As part of #1691, we have removed lighthouse from simorgh in favour of running lighthouse in our CD process. With this we have upgraded to run the latest version of lighthouse which needs Node 10+. Since we are currently locked to Node 8, we cannot run the latest version of lighthouse locally. Once #941 is completed we can re-enable lighthouse to be ran locally.
**Describe the solution you'd like**

- Install Lighthouse (the latest version=5.0.0)
- Add a command to run the lighthouse onto the localhost server

We should be able to run LH against localhost using the same version we use in our CD pipeline. We should remove any audit that forces us to drop the score thresholds due to the report being run against localhost.

We should then try and run this same setup on CI (Travis & Jenkins)

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? No, we are readding an issue. 

**Additional context**

### PR DESCRIPTION
**Pull request by DemonusPC**
Resolves #1694

**Overall change:**

**Code changes:**
- Installed lighthouse
- Added a node script that takes the json report and tests for budgets
- Added the scripts in package.json
- Added lighthouse tests to Travis. 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: scripts/lighthouseBudget.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+
+const getCategoryScores = data => {
+  const { categories } = data;
+
+  return {
+    ally: categories.accessibility.score * 100,
+    bestPractises: categories['best-practices'].score * 100,
```
Comment (alex-magana) [inline scripts/lighthouseBudget.js:9]: MIght it be better to use the dot notation here as well for consistency?

File: scripts/lighthouseBudget.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  result,
+});
+
+const compareToBudget = (categories, scoreResult, scoreBudget) => {
+  let result = true;
+  const logArray = [];
+  // eslint-disable-next-line consistent-return
+  categories.forEach(prop => {
+    const test = isAboveThreshold(scoreResult[prop], scoreBudget[prop]);
+    const b = logRow(prop, scoreResult[prop], scoreBudget[prop], test);
```
Comment (alex-magana) [inline scripts/lighthouseBudget.js]: Please consider using a more descriptive variable name here.

**Review by alex-magana — COMMENTED** [review]
Please have a look at the comments I've added.
Thanks.

File: scripts/lighthouseBudget.js (hunk 1 above)
Comment (DemonusPC) [reply scripts/lighthouseBudget.js:9]: You can't do a dot notation here because of the `-` character

File: scripts/lighthouseBudget.js (hunk 2 above)
Comment (DemonusPC) [reply scripts/lighthouseBudget.js]: Yea good point fixing in now

File: scripts/lighthouseBudget.js (hunk 1 above)
Comment (alex-magana) [reply scripts/lighthouseBudget.js:9]: Pardon me. Sorry that escaped me.

**Review by alex-magana — DISMISSED** [review]

**Review by chris-hinds — APPROVED** [review]

**Review by alex-magana — APPROVED** [review]
Reinstating stale approval.
LGTM.

**Comment by DemonusPC** [comment]
Currently blocked on the simorgh wide licensing issue.

**Pull request merged**